### PR TITLE
Post response if unkown action is provided

### PR DESCRIPTION
**Pull request by hanzei**

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -229,5 +229,7 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 		return &model.CommandResponse{}, nil
 	}
 
+	p.postCommandResponse(args, fmt.Sprintf("Unknown action %v", action))
```
Comment (jasonblais) [inline server/command.go]: Future improvement may be to return the output of `/github help`, so the user can immediately see what commands are available

Comment (hanzei) [reply server/command.go]: 0/5 that just printing "Unknown action" is enough here. Your call.

Comment (jasonblais) [reply server/command.go]: Yes, my note was intended as a future improvement, this PR alone is an improvement (it's better to return a message than no message at all)

**Comment by hanzei** [comment]
@jwilander Gentle reminder to review this

**Comment by jwilander** [comment]
Thanks for the reminder!

**Pull request merged**